client/src/components/SearchAlbums.js

```diff
@@ -1,89 +1,107 @@
-import React from 'react'
```

Comment:
This is where the most merges occurred. I added some things which are necessary for conditional rendering of favorite buttons